core/src/main/java/com/datastrato/gravitino/authorization/AccessControlManager.java

```diff
@@ -101,6 +101,14 @@ public User getUser(String metalake, String user)
     return doWithNonAdminLock(() -> userGroupManager.getUser(metalake, user));
   }
 
+  public String[] listUsers(String metalake) throws NoSuchMetalakeException {
+    return doWithNonAdminLock(() -> userGroupManager.listUsers(metalake));
+  }
+
+  public User[] listUsersInfo(String metalake) throws NoSuchMetalakeException {
+    return doWithNonAdminLock(() -> userGroupManager.listUsersInfo(metalake));
+  }
```

Comment:
I think `listUsers` -> `listUserNames`, `listUsersInfo` -> `listUsers`  would be more appropriate

Reply:
Done.